README.md

```diff
@@ -1,106 +1,72 @@
-		write-through buffer) and the "main-controller" (datapath that asserts ready 
-		when all conditions are met).
-		
-		back_end_native, back_end_axi: Back-End modules, contain both the controllers 
-		write_channel and read_channel respective for the top-level selected.
-		
-		replacement_policy: replacement policy module, only implemented if cache is 
-		set-associative. Currently has LRU, PLRU mru-based, and PLRU (binary) tree-based.
-		
-		cache_control: optional module used for performance measurement (if 
-		counters are implemented), cache invalidation and buffer status.
-		
-	
-	./header/: IOb-Cache's Verilog header source.
-	
-	./simulation/: simulation sources, available for the cache (both native and AXI) 
-	and replacement policy module. Change the header files in this folder to change 
-	the test's settings. Requires Icarus Verilog and GTKwave.
-	
-	./fpga/: synthesis sources, currently setup for Kintex KU040 Ultrascale FPGA
-	
-		synth.tcl: The module for the synthesis can be selected here.
-		
-		synth.xdc: The clock can be changed for different synthesis results (both 
-		resources and timings).
-		
-/software/: Software C programing drivers.
-
-/submodules/: Submodules required for both IOb-Cache (or its benchmarks) and other IObundle 
-projects (IOb-SoC).
-
-
-## Clone the repository
-
-``git clone --recursive git@github.com:IObundle/iob-cache.git``
-
-Access to Github by *ssh* is mandatory so that submodules can be updated.
-
-
+## Cloning the repository
+```
+git clone --recursive git@github.com:IObundle/iob-cache.git
+```
+## Environment setup
 
 ## Simulation
+Simulation supports both Icarus Verilog and Verilator simulators. 
 
-The following commands will run the simulations. Requires Icarus Verilog and GTKWAVE.
-
-To simulate IOb-Cache:
+To simulate, run:
 ```
-make sim
+make sim 
 ```
-Follow by seeing the results using GTKWAVE
+In this case Icarus Verilog is used by default.
 
+To select a specific simulator, run:
 ```
-make gtkwave
+make sim SIMULATOR=<simulator name>
 ```
-
-To simulate Replacement Policy module:
+\<simulator name\> can be icarus or verilator
+For example:
 ```
-make rp_sim
+make sim SIMULATOR=verilator
 ```
-Follow by seeing the results using GTKWAVE
-
+To simulate with generating VCD file using Icarus Verilog, run:
 ```
-make gtkwave_rp
+make sim VCD=1 
 ```
+This command also allows to open gtkwave waveform viewer.
 
-## Synthesis
+To simulate with regression testing, run 
+``` 
+make sim-test
+```
+In this case Icarus Verilog is selected by default. 
+If you are interested in selecting another simulator, set SIMULATOR parameter as explained previously. 
 
-The following commands will run synthesis, requires Vivado (with application open: "source /Vivado's path/setting64.sh").
-Setting can be changed in /hardware/fpga/"board of choice (currently only 1 available)".
+To simulate with regression testing for all simulators, run 
 ```
-make synth
+make test-sim
```

Comment:
I wouldprefer 
make test-sim-all

Reply:
This target does exist.
To implement the last two suggestions, I suggest to rename make targets:
test-sim instead of sim-test
test-sim-all instead test-sim